Writerside/topics/Quickstart-Up-And-Running.md

```diff
@@ -0,0 +1,463 @@
+ec2_ip = "100.28.26.117"
+powershell_deploy_command = "$env:GIT_SSH_COMMAND='ssh -i ~/.ssh/id_rsa -o IdentitiesOnly=yes'; git push dokku main"
+unix_deploy_command = "GIT_SSH_COMMAND='ssh -i ~/.ssh/id_rsa -o IdentitiesOnly=yes' git push dokku main"
+windows_deploy_command = "set \"GIT_SSH_COMMAND=ssh -i ~/.ssh/id_rsa -o IdentitiesOnly=yes\" && git push dokku main"
+```
+
+These outputs will be used to deploy the platform to your newly configured instance.
+
+### Install Catena
+
+Installing Catena is as easy as doing a git push.
+
+Start by cloning the `catena-tools-core` repo:
+
+```bash
+git clone git@github.com:CatenaTools/catena-tools-core.git
+cd catena-tools-core/
+```
+
+Then add a new remote and do a git push, this will deploy the `HEAD` of the repo. 
+
+This will be the value of the `add_doku_remote_command` output above.
+
+```bash
+git remote add dokku dokku@dev.catenatools.com:platform
+git push dokku HEAD:main
+```
+
+Dokku will then show deploy logs:
+
+```
+remote: Resolving deltas: 100% (5066/5066), done.
+-----> Set main as deploy-branch
+-----> Cleaning up...
+-----> Building platform from Dockerfile
+remote: #0 building with "default" instance using docker driver
+remote:
+remote: #1 [internal] load build definition from Dockerfile
+remote: #1 transferring dockerfile: 1.50kB done
+remote: #1 DONE 0.1s
+remote:
+remote: #2 [internal] load metadata for mcr.microsoft.com/dotnet/sdk:8.0
+remote: #2 DONE 0.2s
+remote:
+remote: #3 [internal] load metadata for docker.io/library/debian:latest
+remote: #3 DONE 0.4s
+remote:
+remote: #4 [internal] load .dockerignore
+remote: #4 transferring context: 51B done
+remote: #4 DONE 0.0s
+
+### Truncated for Brevity
+
+=====> Application deployed:
+       http://platform.jlyon1.catenatools.com
+       https://platform.jlyon1.catenatools.com
+
+To jlyon1.catenatools.com:platform
+ * [new branch]      HEAD -> main
+```
+
+Now we're all set! 
+
+> You can update your platform the same way, simply commit the changes and push them to the `main` branch on the dokku remote. The server will automatically build and deploy your changes with no downtime.
+
+
+You can test your platform by logging in:
+
+```Bash
+SESSION_ID=$(curl -D - -sS --location 'https://platform.jlyon1.catenatools.com/api/v1/authentication/login' \
+      --header 'Content-Type: application/json' \
+      --data '{
+      "provider": "PROVIDER_UNSAFE",
+      "payload": "test53"
+  }' | grep "session-*" | cut -d' ' -f2)
+
+curl --location 'https://platform.jlyon1.catenatools.com/api/v1/accounts' \
+--header "session-id: $SESSION_ID" \
+--header 'Content-Type: application/json' \
+--data '{}'
+
+# {"account":{"id":"account-f378e3c2-8760-4059-ba91-69f5343dfb48","displayName":"test53","authRole":"user","metadata":{},"providers":["PROVIDER_UNSAFE"]}}
+```
+
+
+The full postman collection can be found here. Set the http-host environment variable to your domain prior to use.
```

Comment:
Link?

Reply:
This is still TODO since I committed a secret, making a note